tools/README.md

```diff
@@ -118,6 +118,99 @@ The conversion script has a minimum memory requirement of 2X of the size of the
 python3 convert_feat_to_wholegraph.py --dataset-path ogbn-mag240m-2p --node-feat-names paper:feat --low-mem
 ```
 
+## Generate Mitra embeddings for graph data
```

Comment:
Could you provide the context of why use Mitra TFM here? And how Mitra TFM will process the node features briefly?